icu4c/source/i18n/unum.cpp

```diff
@@ -110,8 +112,8 @@ unum_open(  UNumberFormatStyle    style,
         break;
 
     case UNUM_DURATION:
-        retVal = new RuleBasedNumberFormat(URBNF_DURATION, Locale(locale), *status);
-        break;
+         retVal = new RuleBasedNumberFormat(URBNF_DURATION, Locale(locale), *status);
+         break;
```

Comment:
```suggestion
        retVal = new RuleBasedNumberFormat(URBNF_DURATION, Locale(locale), *status);
        break;
```
back to indent 8 not 9